lib/options-from-context.js

```diff
@@ -0,0 +1,17 @@
+// Copyright IBM Corp. 2013,2016. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+module.exports = buildOptionsFromRemotingContext;
+
+function buildOptionsFromRemotingContext(ctx) {
+  var accessToken = ctx.req.accessToken;
+  var options = {
```

Comment:
It would be great if `ctx.options` was extensible through a model or mixin. Eg. being able to implement the constructor of the `options` object.